framework/components/ecl-expandables/ecl-expandables.config.js

```diff
@@ -1,6 +1,16 @@
 module.exports = {
   title: 'Expandables',
   label: 'Expandables',
-  status: 'planned',
+  status: 'ready',
   tags: ['molecule'],
+  context: {
+    _demo: {
+      scripts: `
+        document.getElementById('example-expandable-button-0').addEventListener(
```

Comment:
Please add documentation for the JS part. Even when it's as simple as that, not every one has the time to read the code to understand the mechanics.